### PR TITLE
Luci app mwan3 fixes

### PR DESCRIPTION
**Pull request by feckert**
This PR fixes some minor problems:

* `track_ip` could/should also be set to a dns hostname
* The sorting of the interfaces on the interface page does not make sense
* fixing css styling https://github.com/openwrt/packages/issues/4131

### CODE REVIEW

File: applications/luci-app-mwan3/luasrc/model/cbi/mwan/interfaceconfig.lua [hunk 1]
```diff
@@ -107,9 +107,9 @@ family = mwan_interface:option(ListValue, "family", translate("Internet Protocol
 	family:value("ipv4", translate("IPv4"))
 	family:value("ipv6", translate("IPv6"))
 
-track_ip = mwan_interface:option(DynamicList, "track_ip", translate("Tracking IP"),
-	translate("This IP address will be pinged to dermine if the link is up or down. Leave blank to assume interface is always online"))
-	track_ip.datatype = "ipaddr"
+track_ip = mwan_interface:option(DynamicList, "track_ip", translate("Tracking host"),
+	translate("This host will be pinged to dermine if the link is up or down. Leave blank to assume interface is always online"))
```
Comment (hnyman) [inline applications/luci-app-mwan3/luasrc/model/cbi/mwan/interfaceconfig.lua]: old typo:
dermine --> determine

You are breaking translations in any case, so you might add a hint that both hostname and ip address are valid.

Comment (feckert) [inline applications/luci-app-mwan3/luasrc/model/cbi/mwan/interfaceconfig.lua]: Will fix it and update the translation

**Comment by feckert** [comment]
```
But at the same time I noticed that there is rather expensive looking statement in mwan3:
https://github.com/openwrt/packages/blob/master/net/mwan3/files/lib/mwan3/mwan3.sh#L733
in the iface_status assessment:
ps -w | grep mwan3track | grep -v grep
```
will fix it and send a PR on packages feed in the next days

**Comment by hnyman** [comment]
> do have to add a dependency in the make file if i us pgrep? Or is this a default packages which is always on board?

pgrep is built by default into busybox. It should be in all systems...
```
root@LEDE:~# pgrep
BusyBox v1.26.2 () multi-call binary.
```
And mwan3 already uses pgrep on other places...
E.g.
https://github.com/openwrt/packages/blob/master/net/mwan3/files/lib/mwan3/mwan3.sh#L393
https://github.com/openwrt/packages/blob/master/net/mwan3/files/usr/sbin/mwan3track#L13

**Comment by feckert** [comment]
> mwan3 already uses pgrep on other places... (usually as "pgrep -f")
> E.g.
> https://github.com/openwrt/packages/blob/master/net/mwan3/files/lib/mwan3/mwan3.sh#L393
> https://github.com/openwrt/packages/blob/master/net/mwan3/files/usr/sbin/mwan3track#L13

yes i know i have added this :-). I was not sure if i have to add this to the makefile dependency. But if it is in the default busybox then i will not add it

**Pull request merged**